app/src/main/AndroidManifest.xml

```diff
@@ -85,11 +85,20 @@
 
         <activity
             android:name=".chat.YouTubeActivity"
-            android:theme="@style/Theme.AppCompat.Dialog">
-        </activity>
+            android:theme="@style/Theme.AppCompat.Dialog" />
+
         <meta-data
             android:name="com.google.android.youtube.API_KEY"
-            android:value="${YOUTUBE_API_KEY}"/>
+            android:value="${YOUTUBE_API_KEY}" />
```

Comment:
unnecessary change

Reply:
This is not an unnecessary change, I added an activity and as usual used default refactoring by pressing cmd+option+L in Android Studio. This should have been done in the PR where this change was made, and I think there is no point in sending a separate PR for such spacing/style issues in a line or two.

Reply:
Not true, even though it should have been sent in original PR, stylistic changes not related to PR pollute review area and git blame and commit history

Reply:
@iamareebjamal I would remove this change here then and fix it as part of the refactoring issue already opened.

Reply:
@arundhati24 if a new feature is to be added then it should be added properly rather than refactor it later as it is not dependent on the server neither the hardware so this will be fixed for a long time .

Reply:
@batbrain7 Please read carefully. I just said I will send a new PR fixing these spacing issues as part of refactoring issue already opened by me. And, this I have mentioned for the "unnecessary change". I will be sending a new PR for all spacing issues in some time.

Reply:
And, I would refactor all the code changes that I have made in this PR, as usual.

Reply:
@arundhati24 yes I deleted the comment immediately if you did not check

Reply:
We do get a mail. Please check things before commenting.

Reply:
I am sorry , I will see to it from now